cmd/mr_create.go

```diff
@@ -204,10 +204,16 @@ func runMRCreate(cmd *cobra.Command, args []string) {
 
 	draft, _ := cmd.Flags().GetBool("draft")
 	if draft {
-		isWIP := strings.EqualFold(title[0:4], "wip:")
-		isDraft := strings.EqualFold(title[0:6], "draft:") ||
-			strings.EqualFold(title[0:7], "[draft]") ||
-			strings.EqualFold(title[0:7], "(draft)")
+		var isWIP, isDraft bool
+
+		// Regex could be used, but we have only a limited set of words for
+		// draft mode, so use hardcoded limits
+		if len(title) >= 4 {
+			isWIP = strings.EqualFold(title[0:4], "wip:")
+			isDraft = strings.EqualFold(title[0:6], "draft:") ||
```

Comment:
You moved the crash one line :wink: 

How about adding a small helper function like
```
func hasPrefix(str, prefix string) bool {
    if len(str) < len(prefix) {
        return false;
    }
    return string.EqualFold(str[0:len(prefix)], prefix)
}
```

Reply:
Wow! Goosh! How could I just miss/mix the minimum vs maximum size!
Well.. today has been a really busy day.. should avoid working in many different project's codes at the same time heheh :/.

I was tempted to use the `strings.hasPrefix()` for that... but it's case sensitive. Will follow your suggestion.
Thanks for the help Florian!

Reply:
Looks good to me at a glance, but it has been a busy day here as well, so take that with a grain of salt.

Reply:
:smile: tomorrow I'll recheck the whole thing again.